frontend/components/ActivityPool.vue

```diff
@@ -214,85 +213,91 @@ const itemsJourneyGuide = ref([
-                                    },
-                                    label: {
-                                        class: 'text-text dark:text-white',
-                                    },
-                                    icon: {
-                                        class: 'text-text dark:text-white',
-                                    },
-                                }"
-                            />
+                            <div class="flex sm:pt-1">
+                                <p
+                                    v-tooltip.top="{
+                                        value: activity.name,
+                                        pt: { root: 'font-nunito' },
+                                    }"
+                                    class="w-[98%] overflow-hidden truncate overflow-ellipsis"
+                                    @click="showInfo(activity.id)"
+                                >
+                                    {{ activity.name }}
+                                </p>
+                                <button
+                                    class="pi pi-ellipsis-v"
+                                    aria-haspopup="true"
+                                    aria-controls="overlay_menu"
+                                    @click="toggle"
+                                />
+                            </div>
+                            <div
+                                class="flex items-center pb-4"
+                                @click="showInfo(activity.id)"
+                            >
+                                <SvgClock class="mr-2 h-4 w-4" />
+                                {{
+                                    format(
+                                        parse(
+                                            activity.estimated_duration,
+                                            "HH:mm:ss",
+                                            new Date(),
+                                        ),
+                                        "H:mm",
+                                    )
+                                }}
+                            </div>
+                            <div class="absolute overflow-hidden">
+                                <Menu
+                                    id="overlay_menu"
+                                    ref="menu"
+                                    :model="itemsJourneyGuide"
+                                    class="relative -ml-5 rounded-xl border-2 border-border-light bg-input dark:border-input-dark"
+                                    :popup="true"
+                                    :pt="{
+                                        root: {
+                                            class: 'relative font-nunito bg-input dark:bg-input-dark overflow-hidden',
+                                        },
```

Comment:
Kann man es so machen, dass während man Activities verschiebt (Vom Pool in den Kalender oder innerhalb des Kalenders) -> grabbing Cursor ?
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/5c26a834-4570-4471-a923-3b2e30d0940e)

Reply:
hab so schnell nichts gefunden